Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -379,6 +379,37 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("SetFillColor")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddExpression(
+        "GetFillColorR",
+        _("Filling color red component"),
+        _("Current value for red filling colors."),
```

Comment:
Let's just copy the name in the description to reduce the number of translations to be done:
`Filing color red component`